packages/console/src/components/Tooltip/index.module.scss

```diff
@@ -27,6 +27,16 @@
     top: 0%;
   }
 
+  &.arrowRight::after {
+    top: 50%;
+    left: 100%;
+  }
+
+  &.arrowLeft::after {
+    top: 50%;
+    left: 0% !important;
```

Comment:
We prefer not to use `!important`. I'll take a look if this is necessary and can be avoided